mfai/torch/models/base.py

```diff
@@ -26,3 +26,9 @@ def input_spatial_dims(self) -> Tuple[int, ...]:
         A model supporting 2d tensors should return (2,).
         A model supporting 2d and 3d tensors should return (2, 3).
         """
+
+    def check_required_attributes(self):
```

Comment:
maybe a comment to explain why we do that ? naming is explicit but i never hurts